### DISCUSSION TITLE
How i can set options Fancybox4?

### DISCUSSION THREAD

**Discussion by chernyadyev** [fancyapps/ui#73] [General]
I'm want' unset autofocus, but i can't do it:

```
const f = new Fancybox([{ src: stHtml, type: "html", options: {autoFocus: false} }]);
console.log(f.options.autoFocus); // true
```
What i'm doing wrong?

**Comment by fancyapps** [comment]
Hi,

What do you mean by "i can't do it", what is the outcome and what did you expect? 
Please, use [Playground](https://fancyapps.com/playground) to create a simple demo .

**Reply by chernyadyev** [reply]
https://fancyapps.com/playground/C6 - Here's an example, I want to remove focus from a close button, but the option doesn't apply, screeshot https://ibb.co/kcg6T4s

**Reply by fancyapps** [reply]
So you want the focus to stay where it is? Do you understand that this is a bad practice? https://developer.yoast.com/blog/the-a11y-monthly-making-modals-accessible/ 

Anyway, I`ll fix that, thanks.

**Reply by fancyapps** [reply]
It was fixed in the latest release. But keep in mind that your syntax was not correct, it should be like this:

```JS
new Fancybox([{ src: "Hi", type: "html"}], {placeFocusBack: false, autoFocus: false, trapFocus: false});
```

https://fancyapps.com/playground/Cc

**Reply by chernyadyev** [reply]
Many thanks! It works!
